lib/parse/base.js

```diff
@@ -159,8 +159,6 @@ Base.prototype.getFileSource = function (filename) {
 			header: false,
 			bare: true
 		});
-	} else if (filename.match(this.jsxExtRegEx)) {
```

Comment:
It caused the file to be transpiled, and that's where exceptions about `Illegal import declaration` were raised. Using detective/precinct theres no need for that transpiling